### PR TITLE
feat: add five minutes to feature flags tutorial

### PR DESCRIPTION
**Pull request by beeme1mr**
## This PR

- adds a new tutorial called Five Minutes to Feature Flags

### Notes

Special thanks to @moredip for writing the [original blog](https://blog.thepete.net/blog/2023/03/02/five-minutes-to-feature-flags/). 🙇‍♂️ 

### Follow-up Tasks

- Update the new landing page to point to this tutorial
- Update the [five minutes to feature flags repo](https://github.com/open-feature/five-minutes-to-feature-flags) to include a link to the docs

### CODE REVIEW

File: docs/tutorials/five-minutes-to-feature-flags.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+Hello, world!
+```
+
+Yep, looks good! Go ahead and stop the server using `Ctrl` + `C`.
+
+## With cows, please
+
+Let’s imagine that we’re adding a new, experimental feature to this hello world service. We’re going to upgrade the format of the server’s response, using [cowsay][cowsay]!
+
+However, we’re not 100% sure that this cowsay formatting is going to work out, so for now we’ll protect it behind a conditional:
```
Comment (dyladan) [inline docs/tutorials/five-minutes-to-feature-flags.md:43]: Starting a sentence with a preposition

File: docs/tutorials/five-minutes-to-feature-flags.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+Without a feature flagging provider OpenFeature is pretty pointless - it’ll just return default values. Instead we want to connect our OpenFeature SDK to a full-fledged feature flagging system - a commercial product, an open-source system, or perhaps a custom internal system - so that it can provide flagging decisions from that system.
+
+Connecting OpenFeature to one of these backends is very straightforward, but it does require that we have an actual flagging framework set up. For now, let’s just configure a really, really simple provider that doesn’t need a backend:
+
+```js reference title="04_openfeature_with_provider.js"
+https://github.com/open-feature/five-minutes-to-feature-flags/blob/main/04_openfeature_with_provider.js
+```
+
+This minimalist provider is exactly that - you give it a hard-coded set of feature flag values, and it provides those values via the OpenFeature SDK.
```
Comment (dyladan) [inline docs/tutorials/five-minutes-to-feature-flags.md:106]:
```suggestion
This minimal provider is exactly that - you give it a hard-coded set of feature flag values, and it provides those values via the OpenFeature SDK.
```

Comment (dyladan) [inline docs/tutorials/five-minutes-to-feature-flags.md:106]: https://owl.purdue.edu/owl/general_writing/punctuation/commas/commas_vs_semicolons.html

```suggestion
This minimalist provider is exactly that; you give it a hard-coded set of feature flag values, and it provides those values via the OpenFeature SDK.
```

File: docs/tutorials/five-minutes-to-feature-flags.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+**For example:**
+
+### CloudBees
+
+```js
+import { CloudbeesProvider } from 'cloudbees-openfeature-provider-node';
+
+const appKey = '[YOUR_APP_KEY]';
+OpenFeature.setProvider(await CloudbeesProvider.build(appKey));
+```
```
Comment (dyladan) [inline docs/tutorials/five-minutes-to-feature-flags.md:142]: Collapsable sections probably would be nice here

````suggestion
**For example:**

<details>
<summary>CloudBees</summary>

```js
import { CloudbeesProvider } from 'cloudbees-openfeature-provider-node';

const appKey = '[YOUR_APP_KEY]';
OpenFeature.setProvider(await CloudbeesProvider.build(appKey));
```
</details>
````

<details>
<summary>CloudBees</summary>

```js
import { CloudbeesProvider } from 'cloudbees-openfeature-provider-node';

const appKey = '[YOUR_APP_KEY]';
OpenFeature.setProvider(await CloudbeesProvider.build(appKey));
```
</details>

**Review by dyladan — COMMENTED** [review]
Looks great

File: docs/tutorials/five-minutes-to-feature-flags.md [hunk 4]
```diff
@@ -12,8 +12,8 @@ We’ll be working with a simple [express][express] server, but if you're famili
 Before we jump into the tutorial, let's quickly get everything setup. You'll need [Git](https://git-scm.com/) and [NodeJS](https://nodejs.org/) version 16 or newer. After that, run the following commands from your favorite terminal.
```
Comment (dyladan) [inline docs/tutorials/five-minutes-to-feature-flags.md:12]: its a single command

```suggestion
Before we jump into the tutorial, let's quickly get everything setup. You'll need [Git](https://git-scm.com/) and [NodeJS](https://nodejs.org/) version 16 or newer. After that, run the following command from your favorite terminal.
```

File: docs/tutorials/five-minutes-to-feature-flags.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+
+However, we’re not 100% sure that this cowsay formatting is going to work out, so for now we’ll protect it behind a conditional:
+
+```js reference title="02_basic_flags.js"
+https://github.com/open-feature/five-minutes-to-feature-flags/blob/main/02_basic_flags.js
+```
+
+Now let's start the server with our basic flag configuration by running `node 02_basic_flags.js`. By default, the service continues to work exactly as it did before, but if we change `withCow` to `true` then the response comes in an exciting new format:
+
+:::tip
```
Comment (moredip) [inline docs/tutorials/five-minutes-to-feature-flags.md:51]: I got confused by having this tip immediately follow the sentence saying that the response would be in a new format - I thought for a moment that the tip was what the output should look like. 

Could we put the tip _after_ the sample output?

**Review by moredip — APPROVED** [review]

File: docs/tutorials/five-minutes-to-feature-flags.md [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+
+## Hello, world
+
+Here’s the service we’ll be working on:
+
+```js reference title="01_vanilla.js"
+https://github.com/open-feature/five-minutes-to-feature-flags/blob/main/01_vanilla.js
+```
+
+Pretty much the most basic express server you can imagine - a single endpoint at `/` that returns a plaintext “Hello, world!” response.
```
[the remark this answers is not in the document]
Comment (beeme1mr) [reply docs/tutorials/five-minutes-to-feature-flags.md:28]: In my opinion, I think the tone works for this type of introduction tutorial. I looked into making this more formal and didn't love the results.

**Pull request merged**